README.md

```diff
@@ -93,6 +93,25 @@ We follow the official OASIS specifications in order to provide as much acceptan
 
 ### Security Considerations
 
+These are the TOP OWASP categories of vulnerabilities which potentially affect the CVRF-CSAF-Converter.
+We are omitting those which do not apply (most of them), since it's a plain command-line tool (e.g. authentication failures)
+
+#### [A03:2021 – Injection](https://owasp.org/Top10/A03_2021-Injection/)
+The XML input for the converter is strictly validated for CVRF. The converter rejects invalid inputs.
+
+However, there is a known issue for inserting HTML with code/script, which could be executed by a CSAF consumer: 
+[Encode HTML in JSON output](https://github.com/csaf-tools/CVRF-CSAF-Converter/issues/5)
+
+#### [A06:2021 - Vulnerable and Outdated Components](https://owasp.org/Top10/A06_2021-Vulnerable_and_Outdated_Components/)
+An automated SAST scanning tool like Github Dependabot can be used to scan the libraries for vulnerabilities.
```

Comment:
Maybe, we should set up CodeQL as a GitHub action to ensure that?

Reply:
Sure, set in #91 and passed, have a look.

Then I would rewrite:
"An automated SAST scanning tool like Github Dependabot can be used to scan the libraries for vulnerabilities."
to
"A CodeQL action is set in this project to spot vulnerabilities in 3rd party libraries."

Reply:
Agreed. Please merge #92 before to be able to refer to it with a link.